src/fields/Field.h

```diff
@@ -0,0 +1,34 @@
+#ifndef FIELD_H
+#define FIELD_H
+
+#include <string>
+
+using namespace std;
+
+enum FieldType { INTEGER, FLOAT, TEXT, CHAR, BOOLEAN };
+
+class Field {
+    
+    private:
+        string fieldName;
```

Comment:
I'm assuming we're tackling `string _fieldName`, etc in another issue?
Just mentioning in case it was supposed to be done here :P